### PR TITLE
[Issue #750] Add ASRSnippet tagging.

### PR DESCRIPTION
**Pull request by glogiotatidis**

### CODE REVIEW

**Comment by glogiotatidis** [comment]
image
image

File: snippets/base/templates/base/snippets_custom_name_with_tags.jinja [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<div id="name">
+  <a href="{{ url('admin:base_asrsnippet_change', object_id=obj.id) }}">
+    {{ obj.name }}
+  </a>
+</div>
+<ul class="tags">
+  {% for tag in obj.tags.all().order_by('name') %}
+    <li>
+      <a href="{{ url('admin:base_asrsnippet_changelist') }}?tag={{ tag.name }}" class="tag">
```
Comment (glogiotatidis) [inline snippets/base/templates/base/snippets_custom_name_with_tags.jinja]: note to self: make so it respects existing filtering

Comment (glogiotatidis) [reply snippets/base/templates/base/snippets_custom_name_with_tags.jinja]: Not important atm and we get the same functionality the the list filter on the side 
image

**Comment by glogiotatidis** [comment]
Rebased. Do you have the time to review @pmac?

**Comment by pmac** [comment]
Oh jeez. Sorry I missed this! I'll review today.

**Comment by glogiotatidis** [comment]
Many thanks and don't worry about the delay (you were on pto or at AH most of the time anyway)

**Pull request merged**